otterdog/blueprints/scorecard_integration.yml

```diff
@@ -0,0 +1,56 @@
+id: scorecard-integration
+name: Integrate Scorecard anaylsis
+description: |-
+  Integrate scorecard analysis into the self-service.
+type: scorecard_integration
+config:
+  repo_selector:
+    name_pattern:
+      - sw360
+      - sw360-frontend
+  workflow_name: scorecard.yml
+  workflow_content: |
+    name: Scorecard analysis workflow
+    on:
+      push:
+        branches:
+        - main
+      schedule:
+        # Weekly on Saturdays.
+        - cron: "22 22 * * 2"
+
```

Comment:
something that I was missing in the first iteration of the workflow would be a concurrency setting like that:

```
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```

Reply:
I think that would be quite useful to add for something like that.
Will work on a registry of blueprints that you can also reference instead of copy&pasting them.

Reply:
Done